assets/airports/kmco.json

```diff
@@ -0,0 +1,6135 @@
+                "BITTE": ["BITTE"],
+                "COEDY": ["COEDY", "BUULL", ["HUNKR", "A260-"], ["DOWNO", "A240+"]],
+                "ILEXX": ["ILEXX", ["HUNKR", "A260-"], ["DOWNO", "A240+"]]
+            },
+            "body": [["BRKWL", "A170+|A240-|S280"], ["JAFAR", "A130+|A190-"], ["CRPIT", "A120+|A150-"], "LAMMP"],
+            "rwy": {
+              "KMCO17L": [["^YAGGO", "A60|S210"], "#106"],
+              "KMCO17R": [["^YAGGO", "A60|S210"], "#106"],
+              "KMCO18L": [["^YAGGO", "A60|S210"], "#106"],
+              "KMCO18R": [["^YAGGO", "A60|S210"], "#106"],
+              "KMCO35L": [["WISHH", "A100+|A120-|S250"], "HARAM", ["DORII", "A80+|S220"], ["^BRICE", "A50|S210"], "#186"],
+              "KMCO35R": [["WISHH", "A100+|A120-|S250"], "HARAM", ["DORII", "A80+|S220"], ["^BRICE", "A50|S210"], "#186"],
+              "KMCO36L": [["WISHH", "A100+|A120-|S250"], "HARAM", ["DORII", "A80+|S220"], ["^BRICE", "A50|S210"], "#186"],
+              "KMCO36R": [["WISHH", "A100+|A120-|S250"], "HARAM", ["DORII", "A80+|S220"], ["^BRICE", "A50|S210"], "#186"]
+            },
+            "draw": [
+               ["COEDY*", "BUULL", "HUNKR", "DOWNO", "BRKWL", "JAFAR", "CRPIT", "LAMMP", "WISHH", "HARAM", "DORII", "BRICE"],
+               ["ILEXX*", "HUNKR"],
+               ["BITTE*", "BRKWL"],
+               ["LAMMP", "YAGGO"]
+            ]
+        },
+        "JOKRS2": {
+            "icao": "JOKRS2",
+            "name": "Jokers Two",
+            "entryPoints": {
+              "COREA": ["COREA"],
+              "FERAL": ["FERAL"]
+            },
+            "body": ["SIMIE", "FAZES", "MOANS", "JOKRS"],
+            "rwy": {
+              "KMCO17L": [["TAYYE", "A100+|A120-|S250"], "ARHIP", ["GIRAF", "A80+"], "DVOLA", ["^LYENS", "A50|S220"], "#006"],
+              "KMCO17R": [["TAYYE", "A100+|A120-|S250"], "ARHIP", ["GIRAF", "A80+"], "DVOLA", ["^LYENS", "A50|S220"], "#006"],
+              "KMCO18L": [["TAYYE", "A100+|A120-|S250"], "ARHIP", ["GIRAF", "A80+"], "DVOLA", ["^LYENS", "A50|S220"], "#006"],
+              "KMCO18R": [["TAYYE", "A100+|A120-|S250"], "ARHIP", ["GIRAF", "A80+"], "DVOLA", ["^LYENS", "A50|S220"], "#006"],
+              "KMCO35L": [["^LORRA", "A70|S210"], "#096"],
+              "KMCO35R": [["^LORRA", "A70|S210"], "#096"],
+              "KMCO36L": [["^LORRA", "A70|S210"], "#096"],
+              "KMCO36R": [["^LORRA", "A70|S210"], "#096"]
+            },
+            "draw": [
+                ["COREA*", "SIMIE", "FAZES", "MOANS", "JOKRS", "TAYYE", "ARHIP", "GIRAF", "DVOLA", "LYENS"],
+                ["FERAL*", "SIMIE"],
+                ["JOKRS", "LORRA"]
+            ]
+        },
+        "LEESE3": {
+            "icao": "LEESE3",
+            "name": "Leese Three",
+            "entryPoints": {
+              "OTK": ["OTK"],
+              "SAV": ["SAV", "SHEMP"]
+            },
+            "body": ["OCF", "ALADN"],
+            "rwy": {
+              "KMCO17L": [["SHIMM", "A110|S250"], "LEESE"],
```

Comment:
There are no restrictions at SHIMM as far as I can see so everything in the rwy section can be moved to the body.
Also after LEESE, missing ORL